### PR TITLE
Personal/gankoji/onboarding part one of many

### PR DESCRIPTION
**Pull request by gankoji**
This bash script simply creates the `core/secret` directory and populates it with the necessary files and dummy data to get Core to not fall over on startup. You will still have to add in the credentials manually.

### CODE REVIEW

**Review by mwbrown — APPROVED** [review]

**Comment by ItsMeBrianD** [comment]
This should be good to go once merge conflicts are resolved

**Comment by ItsMeBrianD** [comment]
How does this relate to #406?

**Pull request closed without merging**
